app/views/items/show.html.erb

```diff
@@ -30,20 +30,51 @@
         <div class="card">
           <div class="card-body p-0">
             <table class="table">
-              <thead>
+            <tbody>
               <tr>
-                <th>Value Per Item</th>
-                <th>Distribution Quantity</th>
-                <th>Package Size</th>
+                <th>Base Item</th>
+                <td><%= @item.base_item.name ? @item.base_item.name : '' %></td>
```

Comment:
```suggestion
                <td><%= @item.base_item.name %></td>
```
If you get a `nil` name it should show up as an empty string anyway. Same for any further lines below of the same pattern.